index.html

```diff
@@ -302,6 +302,37 @@ <h4>Text layout</h4>
             the OS/2 table [[OPEN-FONT-FORMAT]] when performing text layout.
           </p>
         </section>
+        <section id="focus">
+            <h4>Focus</h4>
+            <p>
+              MathML provides the ability for authors to provide
```

Comment:
"provide" is repeated twice

Reply:
Yeah, that doesn't look great, even if it is entirely correct -- replaced the second with 'allow for'